python/tvm/topi/hexagon/slice_ops/relu.py

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import tvm
+from tvm import te, tir
+from tvm.ir.module import IRModule
+from tvm.script import tir as T
+
+
+def relu_te_compute(Input, out_shape, dtype):
+    x = tvm.tir.const(0, dtype)
+    Output = te.compute(
+        out_shape, lambda n, h, w, c: tvm.te.max(Input[n, h, w, c], x), name="reluf16"
```

Comment:
Instead of using `out_shape` as an argument to `te.compute`, I'd recommend using `Input.shape`.  That way, the `out_shape` parameter could be removed, and the user wouldn't need to specify it independent of the `Input`.

Reply:
Thank you for pointing this out.